stories/Atom/Navigation/Scroll-bar/ScrollBar.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./scrollbar.scss";
+
+export const ScrollBar = ({}) => (
+  <>
+    <div className="Scrollbar-horizontal-wrapper Scrollbar-horizontal">
+      <div class="Scrollbar-data">.</div>
+    </div>
+    <div className="Scrollbar-vertical-wrapper Scrollbar-vertical">
+      <div class="Scrollbar-data__verticle"></div>
```

Comment:
@harishkumar-srijan Please correct the vertical spelling

Reply:
Done